modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/UserCard.java

```diff
@@ -14,11 +14,24 @@
 
 package com.liferay.frontend.taglib.clay.servlet.taglib.soy;
```

Comment:
Aren't we planning to move these models out of the `.soy` package?

Reply:
How we move it? I mean, if we move it we'll have to change every import in portal. But what if someone is using it from the outside? That's a breaking change, right? (Just to be sure that you mean move and not copy xD)

Reply:
We talked about this in Slack and @izaera validated it.

Create a new one and move all the implementation there, and have this one extend the new one.

Reply:
OK!

Reply:
Just for @julien and @markocikos to know, I'll not be doing this, I'm leaving it as it is right now due to some problems while trying to follow the approach @izaera described in slack

Reply:
I'll create tasks for this in the epic so we can work on it at the very end 👍

Reply:
@carloslancha thanks, I'm not there but I'll keep my eyes on this and the [slack](https://liferay.slack.com/archives/G3JBR21HA/p1602587386397500) conversation.
One step at a time :+1:

Reply:
> I'll create tasks for this in the epic so we can work on it at the very end

Thanks!